modulus/distributed/fft.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from modulus.distributed.manager import DistributedManager
+from modulus.distributed.utils import distributed_transpose, pad_helper, truncate_helper
+from modulus.distributed.mappings import gather_from_parallel_region
+from modulus.distributed.mappings import scatter_to_parallel_region
+
+
+def conj_pad_helper_2d(tensor, pad_dim, other_dim, new_size):
```

Comment:
Yes, that is a good point since other modules also need access to this. But can maybe leave that for a future PR to consolidate these different utilities.